app/components/widgets/forms/time-picker.js

```diff
@@ -28,16 +28,23 @@ export default Component.extend({
       }
     };
 
-    switch (this.rangePosition) {
-      case 'start':
-        defaultOptions.endCalendar = this.$().closest('.fields').find('.ui.calendar.time.picker');
-        break;
-      case 'end':
-        defaultOptions.startCalendar = this.$().closest('.fields').find('.ui.calendar.time.picker');
-        break;
-    }
+    // switch (this.rangePosition) {
+    //   case 'start':
+    //     defaultOptions.endCalendar = this.$().closest('.fields').find('.ui.calendar.time.picker');
+    //     break;
+    //   case 'end':
+    //     defaultOptions.startCalendar = this.$().closest('.fields').find('.ui.calendar.time.picker');
+    //     break;
```

Comment:
@CosmicCoder96 Remove the comments, Else tested every widget works well 🎉

Reply:
leaving these comments.. because someday we might want the time validation to change it's values according to its start counterpart, removed it for now because jquery ui calendar does not compare time while matching fields.